src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java

```diff
@@ -2215,4 +2192,44 @@ private YesNo getYesNoFromString(String yesNoString) {
         }
         return null;
     }
+
+    @Test
+    public void givenNotificationEventTypeAndSenderIsInValid_shouldManageNotificationAndSubscriptionAccordingly() {
+        DynamicList sender = new DynamicList(new DynamicListItem("dwp", "dwp"), new ArrayList<>());
+        SscsCaseData caseData = SscsCaseData.builder().ccdCaseId("1234").originalSender(sender).build();
+        CcdNotificationWrapper notificationWrapper = new CcdNotificationWrapper(
+                SscsCaseDataWrapper.builder().notificationEventType(ACTION_FURTHER_EVIDENCE).newSscsCaseData(caseData).build());
+
+        notificationService.manageNotificationAndSubscription(notificationWrapper, false);
+        verifyExpectedLogMessage(mockAppender, captorLoggingEvent, notificationWrapper.getNewSscsCaseData().getCcdCaseId(),
+                "Incomplete Information with empty or no Information regarding sender for event", Level.INFO);
+    }
+
+    @Test
+    public void isNotificationStillValidToSendSetAsideRequestIsNull() {
+        SscsCaseData caseData = SscsCaseData.builder().ccdCaseId("1234").originalSender(null).build();
+        assertFalse(notificationService.isNotificationStillValidToSendSetAsideRequest(caseData, ACTION_FURTHER_EVIDENCE));
+    }
+
+    @Test
+    public void isNotificationStillValidToSendSetAsideRequestIsNotNull() {
+        DynamicList sender = new DynamicList(new DynamicListItem("dwp", "dwp"), new ArrayList<>());
+        SscsCaseData caseData = SscsCaseData.builder().ccdCaseId("1234").originalSender(sender).build();
+        assertFalse(notificationService.isNotificationStillValidToSendSetAsideRequest(caseData, ACTION_FURTHER_EVIDENCE));
+    }
+
+    @Test
+    public void isNotificationStillValidToSendSetAsideRequestIsNotNull1() {
```

Comment:
Please rename these last two tests it is difficult to understand what is being tested and why the result is different